notificationworkerlambda/cdk/lib/senderworker.ts

```diff
@@ -115,15 +117,35 @@ class SenderWorker extends cdkcore.Construct  {
       reservedConcurrentExecutions: props.reservedConcurrency
     })
 
+    const senderLambdaJar = new lambda.Function(this, `SenderLambdaJar`, {
```

Comment:
We are not allowed to change a lambda from container deployment to JAR deployment (or vice versa)

Here I create a new lambda which will be deployed as JAR, and just change the target lambda of the event source mapping below.   So it will be easy to fall back if we need to use container deployment again.